data/time_series_test.go

```diff
@@ -1009,6 +1016,64 @@ func TestLongToWide(t *testing.T) {
 	}
 }
 
+func TestLongToWideBool(t *testing.T) {
+	tests := []struct {
+		name          string
+		longFrame     *data.Frame
+		wideFrame     *data.Frame
+		tsFillMissing *data.FillMissing
+		Err           require.ErrorAssertionFunc
+	}{
+		{
+			name: "one value, one bool factor",
+			longFrame: data.NewFrame("long_to_wide_test",
+				data.NewField("Time", nil, []time.Time{
+					time.Date(2020, 1, 2, 3, 4, 0, 0, time.UTC),
+					time.Date(2020, 1, 2, 3, 4, 0, 0, time.UTC),
+					time.Date(2020, 1, 2, 3, 4, 30, 0, time.UTC),
+					time.Date(2020, 1, 2, 3, 4, 30, 0, time.UTC),
+				}),
+				data.NewField("Values Floats", nil, []float64{
+					1.0,
+					2.0,
+					3.0,
+					4.0,
+				}),
+				data.NewField("Enabled Factor", nil, []bool{
+					true,
+					false,
+					true,
+					false,
+				})),
+			tsFillMissing: nil,
+			wideFrame: data.NewFrame("long_to_wide_test",
+				data.NewField("Time", nil, []time.Time{
+					time.Date(2020, 1, 2, 3, 4, 0, 0, time.UTC),
+					time.Date(2020, 1, 2, 3, 4, 30, 0, time.UTC),
+				}),
+				data.NewField(`Values Floats`, data.Labels{"Enabled Factor": "false"}, []float64{
+					2.0,
+					4.0,
+				}),
+				data.NewField(`Values Floats`, data.Labels{"Enabled Factor": "true"}, []float64{
+					1.0,
+					3.0,
+				})),
+			Err: require.NoError,
+		},
+	}
+	for i := range tests {
+		tt := tests[i]
+		t.Run(tt.name, func(t *testing.T) {
+			frame, err := data.LongToWide(tt.longFrame, tt.tsFillMissing)
+			tt.Err(t, err)
+			if diff := cmp.Diff(tt.wideFrame, frame, data.FrameTestCompareOptions()...); diff != "" {
+				t.Errorf("Result mismatch (-want +got):\n%s", diff)
+			}
```

Comment:
```suggestion
			diff := cmp.Diff(tt.wideFrame, frame, data.FrameTestCompareOptions()...)
			assert.Emptyf(t, diff, "Result mismatch (-want +got):\n%s", diff)
```

Reply:
I'm not sure I follow the practical difference? If multiple tests fail they currently all show up there diffs. This change seems to make for more independent output that isn't as easy to read it seems.

Reply:
But actually this is a repeatted pattern, so if we want to change it, would rather do it in another PR to keep things matched up.

Reply:
@kylebrandt Ah, I just assumed `t.Errorf` would exit the test, but maybe that's `t.Fatalf`.